### PR TITLE
IPoMPLS: clarifications, proposals for less figures

### PR DESCRIPTION
**Pull request by vb-balazs**

### CODE REVIEW

**Review by dfedyk — APPROVED** [review]
Looks good to me.

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 1]
```diff
@@ -261,9 +264,12 @@ End System        Node         Node           Node        End System
                         [Network]                   [Network]
                          `-----'                     `-----'
 
-        |<- DN IP->| |<---- DetNet MPLS ---->| |< -DN IP ->|
+                     |<---- DetNet MPLS ---->|
+        |<--------------------- DetNet IP ------------------->|
         ]]></artwork>
         </figure>
+        <t> [Editor's note: Fig1 and Fig2 are identical. Fig2 to be deleted?]
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:271]: Move T.Pe from fig2 to fig1 and keep fig1.

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 2]
```diff
@@ -241,7 +244,7 @@
       <section title="IP Over DetNet MPLS Data Plane Scenarios"
                anchor="sec_ip_mpls_dt_dp_scen">
         <t>
-          An example use of IP over DetNet MPLS follows below.
+          An example use of DetNet IP over DetNet MPLS follows below.
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:247]: Do sub-sections for IP over DetNet MPLS and DetNet IP over Detnet MPLS.

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 3]
```diff
@@ -397,26 +410,31 @@ System  |    +--------+       +--------+       +--------+   | System
...
+           /->     +------+  +------+  +------+            ^ ^
+           |       |  X   |  |  X   |  |  X   |<- App-Flow : :
+           |       +------+  +------+  +------+            : :
+App-Flow <-+       |NProto|  |NProto|  |NProto|            : :(1)
+ for MPLS  |       +------+  +------+  +------+            : :
+           |       |  IP  |  |  IP  |  |  IP  |            : v
+           \-> +---+======+--+======+--+======+-----+      :
+DetNet-MPLS        | d-CW |  | d-CW |  | d-CW |            : 
+                   +------+  +------+  +------+            :(2)
+                   |Labels|  |Labels|  |Labels|            v
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:435]: Drop NProto and X from the figure.

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 4]
```diff
@@ -397,26 +410,31 @@ System  |    +--------+       +--------+       +--------+   | System
...
+App-Flow <-+       |NProto|  |NProto|  |NProto|            : :(1)
+ for MPLS  |       +------+  +------+  +------+            : :
+           |       |  IP  |  |  IP  |  |  IP  |            : v
+           \-> +---+======+--+======+--+======+-----+      :
+DetNet-MPLS        | d-CW |  | d-CW |  | d-CW |            : 
+                   +------+  +------+  +------+            :(2)
+                   |Labels|  |Labels|  |Labels|            v 
                +---+======+--+======+--+======+-----+
-Sub-Network        |  L2  |  | TSN  |  | UDP  |
+Link/Sub-Network   |  L2  |  | TSN  |  | UDP  |
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:437]: Make consistent with the Framework.

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 5]
```diff
@@ -428,7 +446,7 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
         </artwork>
       </figure>
       <t>
-        In the figure, "IP App-Flow" indicates the payload carried by
+        In the figure, "App-Flow" indicates the payload carried by
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:449]: Rerfer to "IP flow" as figure changed to "IP"

File: ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml [hunk 6]
```diff
@@ -460,26 +478,34 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
...
+      <t>
+        [Editor's note: several proposed changes to clarify refered 
+		components. Confusing usage of app-flow terminology.]
+      </t>
+
         <t>
-	  A relay node that sends a DetNet IP flow over a DetNet MPLS network
-	  MUST map that single DetNet IP flow into a single app-flow and MUST
-	  process that app-flow in accordance to the procedures defined in
+	  A DetNet relay node (ingress T-PE) that sends a DetNet IP flow over a DetNet MPLS network
```
Comment (vb-balazs) [inline ip-over-mpls/draft-ietf-detnet-ip-over-mpls.xml:493]: Re-read this section again.

**Pull request merged**